src/main/java/sirius/biz/storage/util/MissingBlobObjectCheckJob.java

```diff
@@ -173,9 +183,7 @@ private synchronized void writeLine(Object... columns) throws IOException {
     }
 
     private synchronized void incrementCounter(String counter) {
-        if (TaskContext.get().isActive()) {
-            process.incCounter(counter);
-        }
+        counters.put(counter, counters.get(counter) + 1);
```

Comment:
Would it make sense to extend the jobs framework to allow for manually displaying counter values in the side bar?

Reply:
Not a bad idea. I will write down in my list to see how easily that can be achieved (just not for this job though)